websites/H/Hexa Watch/presence.ts

```diff
@@ -0,0 +1,60 @@
+const presence = new Presence({
+  clientId: '1370078552581603489',
+});
+
+let startTimestamp = Math.floor(Date.now() / 1000);
+
+enum ActivityAssets {
+  Logo = 'https://cdn.rcd.gg/PreMiD/websites/H/HexaWatch/assets/logo.png',
+}
+
+presence.on('UpdateData', async () => {
+  const { pathname } = document.location;
+  const pathList = pathname.split('/').filter(Boolean);
+
+  const presenceData: PresenceData = {
+    largeImageKey: ActivityAssets.Logo,
+    buttons: [
+      { label: 'Open Hexa Watch', url: 'https://hexa.watch' },
+    ],
```

Comment:
[Buttons](https://github.com/PreMiD/Activities/blob/main/.github/CONTRIBUTING.md#presencets) which lead to the home page are not allowed.
```suggestion
```